hieroplatform/src/main/java/org/hiero/table/api/IIntColumn.java

```diff
@@ -36,6 +36,11 @@ default String asString(final int rowIndex) {
         return Integer.toString(this.getInt(rowIndex));
     }
 
+    /**
+     * Given values x and y indexed by i and j respectively,  return sign(x - y) in {-1, 0, 1}
```

Comment:
Let x and y be values given by the indexes i and j.
You can move part of this comment in the `@return` line.